### PR TITLE
Improve access credential handling

### PR DESCRIPTION
**Pull request by saxix**
make user/password optional argument:

read DEVPI_USER/DEVPI_PASS environment variables or prompt for them if no values provied

### CODE REVIEW

**Comment by saxix** [comment]
I'm going to update tests/README if you like the PR

File: setup.cfg [hunk 1]
```diff
@@ -2,7 +2,7 @@
 universal = 1
 
 [tool:pytest]
-addopts = --cov-report= --cov=devpi_builder tests
+;addopts = --cov-report= --cov=devpi_builder tests
```
Comment (theMarix) [inline setup.cfg]: As far as I can see this breaks the coverage analysis in the Travis build. I assume you changed this to be able to run without coverage locally. Please either re-add or explicitly add the arguments to the test invocation in the Travis build.

FYI: You can also invoke py.test with `--no-cov` if you locally want to run without coverage.

File: devpi_builder/cli.py [hunk 2]
```diff
@@ -110,6 +116,11 @@ def main(args=None):
     parser.add_argument('--client-cert', help='Client key to use to authenticate with the devpi server.', default=None)
 
     args = parser.parse_args(args=args)
+    if args.user is None:
+        args.user = input('Username: ')
```
Comment (saxix) [inline devpi_builder/cli.py]: `DEVPI_USER="" DEVPI_PASSWORD="" devpi-builder ..... `
will skip interactive prompt.
Let me know if you prefer add '--batch' or '--no-input'.

File: setup.cfg (hunk 1 above)
Comment (saxix) [inline setup.cfg]: ops that was my mistake....

File: devpi_builder/cli.py (hunk 2 above)
Comment (theMarix) [inline devpi_builder/cli.py]: I'd prefer the `--batch` naming as I know more tools using this.

**Pull request merged**